src/status_im/node/models.cljs

```diff
@@ -49,6 +50,6 @@
 
 (defn initialize
   [address {{:keys [status-node-started?] :as db} :db :as cofx}]
-  (if (not status-node-started?)
-    (start address cofx)
-    (restart)))
+  (if (or status-node-started? platform/desktop?)
```

Comment:
@siphiuel, don't we have `status-node-started?` correctly set when running desktop build?

Reply:
Good question! Just checked - we have it set during the first run, but if we e.g. `Reload JS`, it becomes `nil`.